File: benchmark/corpus/18_correct_but_refused/b_euclid.py
# The control: the greatest common divisor by Euclid's algorithm, a loop with no counter that still always ends.
def gcd(a: int, b: int) -> int:
    x, y = a, b
    while y != 0:
        x, y = y, x % y
    return x


print("gcd " + str(gcd(1071, 462)))

File: benchmark/corpus/18_correct_but_refused/b_euclid.js
// The control: the greatest common divisor by Euclid's algorithm, a loop with no counter that still always ends.
function gcd(a, b) {
  let x = a;
  let y = b;
  while (y !== 0) {
    const t = y;
    y = x % y;
    x = t;
  }
  return x;
}

console.log("gcd " + gcd(1071, 462));
